pkg/specgen/specgen.go

```diff
@@ -539,18 +539,18 @@ type ContainerNetworkConfig struct {
 	// Conflicts with HostAdd.
 	// Optional.
 	UseImageHosts *bool `json:"use_image_hosts,omitempty"`
-	// BaseHostsFile is the path to a hosts file, the entries from this file
-	// are added to the containers hosts file. As special value "image" is
-	// allowed which uses the /etc/hosts file from within the image and "none"
-	// which uses no base file at all. If it is empty we should default
-	// to the base_hosts_file configuration in containers.conf.
-	// Optional.
-	BaseHostsFile string `json:"base_hosts_file,omitempty"`
 	// HostAdd is a set of hosts which will be added to the container's
 	// /etc/hosts file.
 	// Conflicts with UseImageHosts.
 	// Optional.
 	HostAdd []string `json:"hostadd,omitempty"`
+	// HostsFile is the base file to create the `/etc/hosts` file inside the infra container.
+	// This must either be an absolute path to a file on the host system, or one of the
+	// special flags `image` or `none`.
+	// If it is empty it defaults to the base_hosts_file configuration in containers.conf.
+	// Conflicts with UseImageHosts.
+	// Optional.
+	HostsFile string `json:"hosts_file,omitempty"`
```

Comment:
Here as well, this is a breaking change as specgen is a stable type exposed in the bindings.

Reply:
This is reverted too